src/main/java/vazkii/botania/api/subtile/SubTileGenerating.java

```diff
@@ -184,7 +179,10 @@ public void readFromPacketNBT(NBTTagCompound cmp) {
 		int y = cmp.getInteger(TAG_COLLECTOR_Y);
 		int z = cmp.getInteger(TAG_COLLECTOR_Z);
 
-		cachedCollectorCoordinates = new ChunkCoordinates(x, y, z);
+    
+		cachedCollectorCoordinates = (x!= 0 && y != -1 && z != 0) ? new ChunkCoordinates(x, y, z) : null;
```

Comment:
yes, they can but not the combination of the 3 x/y/z.

Its just a simple check to prevent loading to the cachedCollector the coordinates saved in the NBT when the linkedCollector is null.